apiController.js

```diff
@@ -86,10 +88,35 @@ exports.postComment = (req, res, next) => {
   }
 
   Promise.all(promises)
-  .then(([post]) => { {
-    res.status(201).send(post);}
-  })
-  .catch((err) => {
-    next(err);
-  });
+    .then(([post]) => {
+      {
+        res.status(201).send(post);
+      }
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.patchVotes = (req, res, next) => {
+  const { article_id } = req.params;
+  const { inc_votes } = req.body;
+
+  if (isNaN(inc_votes)) {
+    return res.status(400).send({ msg: "bad request: not a valid vote input, try again" });
+  }
+
+  if (inc_votes < 0) {
+    return votePatchMinus(article_id, inc_votes).then((article) => {
+      res.status(200).send(article);
+    });
+  }
+
+  return votePatchAdd(article_id, inc_votes)
```

Comment:
There is no need for 2 models the same model is able to add both negative and positive votes to the existing ones (maths does the works for us!)